src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -69,8 +69,19 @@ const showUserDetails = (accountID) => {
 function ReportActionItemSingle(props) {
     const actorEmail = lodashGet(props.action, 'actorEmail', '').replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
     const actorAccountID = props.action.actorAccountID;
-    const {avatar, displayName, pendingFields} = props.personalDetailsList[actorAccountID] || {};
-    const avatarSource = UserUtils.getAvatar(avatar, actorAccountID);
+    let {avatar, displayName} = props.personalDetailsList[actorAccountID] || {};
+    const {pendingFields} = props.personalDetailsList[actorAccountID] || {};
```

Comment:
this can be combined into a single line?

```suggestion
    let {avatar, displayName, pendingFields = {} } = props.personalDetailsList[actorAccountID] || {};
```

although `pendingFields` will be declared with `let` in this case.

Reply:
(It will be a linter error)